ankimorphs/text_preprocessing.py

```diff
@@ -10,6 +10,7 @@
 round_brackets_regex = re.compile(r"（[^）]*）")
 slim_round_brackets_regexp = re.compile(r"\([^)]*\)")
 non_alpha_regexp = re.compile(r"[-'\w]")
+ruby_regex = re.compile(r" ?([^] ]+?)\[(.+?)\]")
```

Comment:
I did make a modification here, but it was not strictly necessary. the `>` was from a prior iteration of the code. this now better reflects what we would like to do. even though this will not be used onece to find all matches, but instead will be looped over, this WOULD now work in a single pass.